addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComWindMessage.java

```diff
@@ -239,8 +217,7 @@ public void setDeviceId(String deviceId) throws RFXComException {
     }
 
     @Override
-    public void convertFromState(RFXComValueSelector valueSelector, Type type) throws RFXComException {
-
+    public void convertFromState(String channelId, Type type) throws RFXComException {
         throw new RFXComException("Not supported");

```

Comment:
Please also make this exception specific (maybe an UnsupportedOperationException) so it does no longer have to be `throws RFXComException` in the abstract class or interface.